packages/core/admin/admin/src/content-manager/pages/EditView/Header/index.js

```diff
@@ -204,6 +206,63 @@ const Header = ({
           />
         </Dialog>
       )}
+      {showPublishConfirmation && (
+        <Dialog
+          onClose={onPublishPromptDismissal}
+          title="Confirmation"
+          labelledBy="confirmation"
+          describedBy="confirm-description"
+          isOpen={showPublishConfirmation}
+        >
+          <DialogBody icon={<ExclamationMarkCircle />}>
+            <Stack spacing={2}>
+              <Flex justifyContent="center" style={{ textAlign: 'center' }}>
+                <Typography id="confirm-description">
+                  {draftCount}
```

Comment:
Do we need to render the count independently from the translation message? The benefit of having it only part of the translation message would be, that the number can actually moved to different positions in the string, if some languages require that.